<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screen_offClass</class>
 <widget class="QDialog" name="screen_offClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>screen_off</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="m_mintorState">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>空闲时间：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_idleTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_isAutoRun">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开机自启动</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="screen_off.qrc"/>
 </resources>
 <connections/>
</ui>
